third_party/bigframes_vendored/pandas/core/generic.py

```diff
@@ -627,6 +627,48 @@ def copy(self):
     def ffill(self, *, limit: Optional[int] = None):
         """Fill NA/NaN values by propagating the last valid observation to next valid.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> import numpy as np
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.DataFrame([[np.nan, 2, np.nan, 0],
+            ...                     [3, 4, np.nan, 1],
+            ...                     [np.nan, np.nan, np.nan, np.nan],
+            ...                     [np.nan, 3, np.nan, 4]],
+            ...                    columns=list("ABCD")).astype("Float64")
+            >>> df
+                A     B     C     D
```

Comment:
[nit] indentation mis-aligned (usually it is right justified), same for other dataframe reprs below

Reply:
Good catch! thanks.